packages/web/server/respondToError.ts

```diff
@@ -0,0 +1,11 @@
+interface Response {
+  status: (code: number) => Response
+  json: (jsonCompatible: object) => void
+}
+
+export default function respondToError(
```

Comment:
Nit: This name makes me think it's a handler for when an error happens but looks like it's actually sending the error response itself. Maybe just `respondError`?